README.md

```diff
@@ -245,8 +245,8 @@ kt_javac_options(
 kt_jvm_library(
     name = "package_name",
     srcs = glob(["*.kt"]),
-    kotlinc_options = "//:kt_kotlinc_options_for_package_name",
-    javac_options = "//:kt_javac_options_for_package_name",
+    kotlinc_opts = "//:kt_kotlinc_options_for_package_name",
+    javac_opts = "//:kt_javac_options_for_package_name",
```

Comment:
According to https://github.com/bazelbuild/rules_kotlin/blob/ebef8c560a0faf29eed54810df084b30c32c6e6e/kotlin/internal/jvm/jvm.bzl#L217-L230.